app/components/countries/CountriesFilters.js

```diff
@@ -4,10 +4,12 @@ import NavLink from 'containers/common/NavLink';
 function TableFilters(props, context) {
   return (
     <div className="row c-table-filters">
-      <div className="column small-12 medium-5 tags">
-        <NavLink to={`/countries/${props.country}/sites`} i18nText="sites" />
-        <NavLink to={`/countries/${props.country}/species`} i18nText="species" />
-        <NavLink to={`/countries/${props.country}/populations`} i18nText="populations" />
+      <div className="column small-12 medium-5">
+        <div className="tags">
+          <NavLink to={`/countries/${props.country}/sites`} i18nText="sites" className={props.category && props.category === 'sites' ? 'is-active' : ''} />
```

Comment:
I think it is better to keep the logic here and send only the class to the NavLink comp.